lms/djangoapps/edxnotes/helpers.py

```diff
@@ -239,39 +244,80 @@ def get_index(usage_key, children):
     return children.index(usage_key)
 
 
-def search(user, course, query_string):
+def construct_urls(request, course_id, api_next_url, api_previous_url):
     """
-    Returns search results for the `query_string(str)`.
+    Construct next and previous urls.
+
+    Arguments:
+        course_id: course id
+        api_next_url: notes api next url
+        api_previous_url: notes api previous url
+
+    Returns:
+        next_url: lms notes next url
+        previous_url: lms notes previous url
     """
-    response = send_request(user, course.id, "search", query_string)
-    try:
-        content = json.loads(response.content)
-        collection = content["rows"]
-    except (ValueError, KeyError):
-        log.warning("invalid JSON: %s", response.content)
-        raise EdxNotesParseError(_("Server error. Please try again in a few minutes."))
-
-    content.update({
-        "rows": preprocess_collection(user, course, collection)
-    })
+    def encoded_params(url):
+        """
+        Extract and return encoded params
+        """
+        keys = ('page', 'page_size', 'text')
+        parsed = urlparse.urlparse(url)
+        query_params = urlparse.parse_qs(parsed.query)
+
+        return urlencode({key: query_params.get(key)[0] for key in keys if key in query_params})
+
+    base_url = reverse("notes", kwargs={"course_id": course_id})
+    next_url = api_next_url
+    previous_url = api_previous_url
+
+    if request.is_secure():
+        host = 'https://' + request.get_host()
+    else:
+        host = 'http://' + request.get_host()
+
+    if next_url:
+        next_url = '{host}{base_url}?{query_params}'.format(
+            host=host,
+            base_url=base_url,
+            query_params=encoded_params(next_url)
+        )
 
-    return json.dumps(content, cls=NoteJSONEncoder)
+    if previous_url:
+        previous_url = '{host}{base_url}?{query_params}'.format(
+            host=host,
+            base_url=base_url,
+            query_params=encoded_params(previous_url)
+        )
+
+    return next_url, previous_url
 
 
-def get_notes(user, course):
+def get_notes(request, course, path='annotations', page=1, page_size=10, query_string=None):
```

Comment:
Can we use `DEFAULT_PAGE` and `DEFAULT_PAGE_SIZE` instead of hardcoding the values.